tests/cassettes/TestCarrierAccountAll.yaml

```diff
@@ -5,51 +5,26 @@ interactions:
-      Account Number","value":"A1A1A1"}}},"credentials":{"account_number":"A1A1A1"},"test_credentials":null},{"id":"ca_2dbd13dff1f14fdebb42c7a9b5f45b2b","object":"CarrierAccount","type":"UpsAccount","clone":false,"created_at":"2022-03-03T02:38:06Z","updated_at":"2022-03-03T02:38:06Z","description":"UPS
-      Account","reference":null,"readable":"UPS","logo":null,"fields":{"credentials":{"account_number":{"visibility":"readonly","label":"UPS
-      Account Number","value":"A1A1A1"}}},"credentials":{"account_number":"A1A1A1"},"test_credentials":null},{"id":"ca_ee74825f4fb64de6b2d2782df384b4a8","object":"CarrierAccount","type":"UpsAccount","clone":false,"created_at":"2022-03-03T02:38:07Z","updated_at":"2022-03-03T02:38:07Z","description":"UPS
-      Account","reference":null,"readable":"UPS","logo":null,"fields":{"credentials":{"account_number":{"visibility":"readonly","label":"UPS
-      Account Number","value":"A1A1A1"}}},"credentials":{"account_number":"A1A1A1"},"test_credentials":null},{"id":"ca_82e3beda2f6641299205a7d1779fc7ad","object":"CarrierAccount","type":"UpsAccount","clone":false,"created_at":"2022-03-03T02:38:07Z","updated_at":"2022-03-03T02:38:08Z","description":"My
-      custom description","reference":null,"readable":"UPS","logo":null,"fields":{"credentials":{"account_number":{"visibility":"readonly","label":"UPS
-      Account Number","value":"A1A1A1"}}},"credentials":{"account_number":"A1A1A1"},"test_credentials":null},{"id":"ca_8bc84350a16c4a6a9344b362aa5b315b","object":"CarrierAccount","type":"UpsAccount","clone":false,"created_at":"2022-03-03T02:44:48Z","updated_at":"2022-03-03T02:44:48Z","description":"UPS
-      Account","reference":null,"readable":"UPS","logo":null,"fields":{"credentials":{"account_number":{"visibility":"readonly","label":"UPS
-      Account Number","value":"A1A1A1"}}},"credentials":{"account_number":"A1A1A1"},"test_credentials":null},{"id":"ca_9bf8286878e8466ba03a563a5f2db15a","object":"CarrierAccount","type":"UpsAccount","clone":false,"created_at":"2022-03-03T02:44:49Z","updated_at":"2022-03-03T02:44:49Z","description":"UPS
-      Account","reference":null,"readable":"UPS","logo":null,"fields":{"credentials":{"account_number":{"visibility":"readonly","label":"UPS
-      Account Number","value":"A1A1A1"}}},"credentials":{"account_number":"A1A1A1"},"test_credentials":null},{"id":"ca_0ff1a988c20442b5aa6246d4356ecb99","object":"CarrierAccount","type":"UpsAccount","clone":false,"created_at":"2022-03-03T02:44:49Z","updated_at":"2022-03-03T02:44:50Z","description":"My
-      custom description","reference":null,"readable":"UPS","logo":null,"fields":{"credentials":{"account_number":{"visibility":"readonly","label":"UPS
-      Account Number","value":"A1A1A1"}}},"credentials":{"account_number":"A1A1A1"},"test_credentials":null}]'
+    body: '[{"billing_type":"easypost","clone":false,"created_at":"2020-05-07T23:43:48Z","credentials":{},"description":"USPS
```

Comment:
It appears that nested objects in a list do not have the correct key ordering here.

Reply:
correct, this is only works with non-array objects string.